orchagent/vnetorch.cpp

```diff
@@ -469,18 +476,115 @@ VnetBridgeInfo VNetBitmapObject::getBridgeInfoByVni(uint32_t vni, string tunnelN
         throw std::runtime_error("vni creation failed");
     }
 
-    tunnel->addEncapMapperEntry(info.bridge_id, vni);
+    // FIXME: Use "createVxlanTunnelMap()" for tunnel mapper creation
+    auto tunnelEncapMapperEntry = tunnel->addEncapMapperEntry(info.bridge_id, vni);
```

Comment:
It is better to call the orch function instead of directly invoking the addEncapMapper and insertMapperEntry. Do you want to fix it in this PR?

Reply:
I agree, but currently it cannot be used due to some issue in MLNX SAI. That is why only "encap" mapper is created directly. So, I added a comment here and it will be fixed later on.